src/python/pants/backend/java/test/junit_test.py

```diff
@@ -109,9 +109,15 @@ def test_vintage_simple_success(rule_runner: RuleRunner) -> None:
             "coursier_resolve.lockfile": JUNIT4_RESOLVED_LOCKFILE.to_json().decode("utf-8"),
             "BUILD": dedent(
                 """\
+                jvm_artifact(
+                  name = 'junit_junit',
```

Comment:
Why this target name? Because it's a target name, it should be able to be anything you want as long as unique. `junit` seems simpler.

Reply:
The general convention in naming targets like this is `GROUP_ARTIFACT` so with group=junit and artifact=junit you get `junit_junit`. I'd rather not deviate from the convention for this target.